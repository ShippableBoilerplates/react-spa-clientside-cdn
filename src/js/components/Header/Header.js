import React, {PureComponent} from 'react'
import {header, title} from './Header.styles'

export default class Header extends PureComponent {
    render() {
        return (
            <div {...header}>
                <h1 {...title}>Ship a React SPA!</h1>
            </div>
        )
    }
}
