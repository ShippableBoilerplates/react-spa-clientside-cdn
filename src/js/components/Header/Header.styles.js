import {css} from 'glamor'

export const header = css({
    height: '10vh',
    background: '#4b586e',
    display: 'flex',
    alignItems: 'center',
    justifyContent: 'center'
})

export const title = css({
    margin: '0',
    color: '#DDDDDD',
    fontWeight: 700,
    fontSize: '2rem'
})
