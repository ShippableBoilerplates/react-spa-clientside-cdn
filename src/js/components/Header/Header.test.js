import ReactShallowRenderer from 'react-test-renderer/shallow'
import React from 'React'
import Header from './Header'

test('should render Header correctly', () => {
    const renderer = new ReactShallowRenderer()
    renderer.render(<Header />)
    expect(renderer.getRenderOutput()).toMatchSnapshot()
})

/*
    Enzyme
*/
