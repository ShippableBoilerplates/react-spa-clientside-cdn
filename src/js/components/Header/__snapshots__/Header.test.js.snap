// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<div
  data-css-p5ucwd=""
>
  <h1
    data-css-7u1zcn=""
  >
    Ship a React SPA!
  </h1>
</div>
`;
